spec/stellar_app/app/services/get_max_amount.rb

```diff
@@ -0,0 +1,7 @@
+class GetMaxAmount
+
+  def self.call
+    1
```

Comment:
Maybe we can put it into a constant and refer to it? I'm not sure what `1` means :)

Reply:
Added a new commit to address this